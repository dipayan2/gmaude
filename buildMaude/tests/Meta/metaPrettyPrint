#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/metaPrettyPrint.maude -no-banner -no-advise \
  > metaPrettyPrint.out 2>&1

diff $srcdir/metaPrettyPrint.expected metaPrettyPrint.out > /dev/null 2>&1
