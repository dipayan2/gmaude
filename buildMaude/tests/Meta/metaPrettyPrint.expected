==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'FLOAT .
  sorts none .
  none
  none
  none
  none
endfm, '_+_['1.0.FiniteFloat, '1.0.FiniteFloat]) .
rewrites: 2
result NeQidList: '1.0 '+ '1.0
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'FLOAT .
  sorts none .
  none
  none
  none
  none
endfm, 'X:Float) .
rewrites: 2
result Variable: 'X:Float

r g b
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, 's_['0.Zero]) .
rewrites: 2
result Sort: '1
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, 's_['s_['0.Zero]], mixfix flat format number rat) .
rewrites: 2
result NeTypeList: 's '1
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, 's_['s_['0.Zero]], none) .
rewrites: 2
result NeTypeList: 's_ '`( 's_ '`( '0 '`) '`)
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '-_['s_['0.Zero]]) .
rewrites: 2
result Sort: '-1
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '_/_['-_['s_['0.Zero]], 's_['0.Zero]]) .
rewrites: 2
result Sort: '-1/1
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '-_['s_^22['0.Zero]]) .
rewrites: 2
result Sort: '-22
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '_/_['-_['s_^6789['0.Zero]], 's_^12345['0.Zero]]) .
rewrites: 2
result Sort: '-6789/12345
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '_/_['-_['s_^6789['0.Zero]], 's_^12345['0.Zero]], mixfix) .
rewrites: 2
result NeTypeList: '- 's_^6789 '`( '0 '`) '/ 's_^12345 '`( '0 '`)
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '_/_['-_['s_^6789['0.Zero]], 's_^12345['0.Zero]], mixfix number) .
rewrites: 2
result NeTypeList: '-6789 '/ '12345
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'RAT .
  sorts none .
  none
  none
  none
  none
endfm, '_/_['-_['s_^6789['0.Zero]], 's_^12345['0.Zero]], mixfix rat) .
rewrites: 2
result Sort: '-6789/12345
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'FLOAT .
  sorts none .
  none
  none
  none
  none
endfm, 'X:Float ; 'Y:Float, 'X:Float, mixfix) .
rewrites: 2
result Sort: 'X
==========================================
reduce in META-LEVEL : metaPrettyPrint(fmod 'FOO is
  protecting 'FLOAT .
  sorts none .
  none
  none
  none
  none
endfm, 'X:Float ; 'Y:Float, '_/_['X:Float, 'Y:Float], mixfix) .
rewrites: 2
result NeTypeList: 'X '/ 'Y
Bye.
